ros/src/computing/planning/decision/packages/decision_maker/include/decision_maker_node.hpp

```diff
@@ -107,6 +115,23 @@ class DecisionMakerNode
   std::string TextOffset;
   std::vector<CrossRoadArea> intersects;
   double displacement_from_path_;
+  autoware_msgs::waypoint CurrentStoplineTarget_;
+
+  bool foundOtherVehicleForIntersectionStop_; // In fact this should be defined as state.
+  class DetectionArea
+  {
+    public:
+      double x1,x2;
+      double y1,y2;
+
+      DetectionArea(){ //need to change to setting by param
+	      x1 =  80.0;
+	      x2 =   0.0;
+	      y1 =  50.0;
+	      y2 = -50.0;
```

Comment:
@yk-fujii I did this here: https://github.com/CPFL/Autoware/pull/1008/commits/6227d56a457a49bb9d74720de75fcdc5abc11b8d

Can you check if it is correct?

Reply:
Thanks, That's correct!